modules/javafx.controls/src/main/resources/com/sun/javafx/scene/control/skin/caspian/caspian.css

```diff
@@ -1505,32 +1505,32 @@
     -fx-background-color: derive(-fx-control-inner-background,-5%);
 }
 
-.list-view:focused > .virtual-flow > .clipped-container > .sheet > .list-cell:focused {
+.list-view:focused > .virtual-flow > .viewport > .clipped-container > .sheet > .list-cell:focused {
```

Comment:
This might require a CSR as it might break custom stylesheets that modify list/tree/table/views.

Reply:
Probably. Was not sure since this is a very specific case which we never had before (as far as I can tell).

Reply:
Yes, I think a CSR would be in order. Thanks for bringing this up @andy-goryachev-oracle